packages/react-components/src/components/NumberField.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import {State, FieldJson} from '@aemforms/af-core';
+import '../../dist/theme.css';
+
+const NumberField = (fieldjson: State<FieldJson>) => {
+   const [props, handlers] = useRuleEngine(fieldjson);
+   const i18n = useFormIntl();
+   const { id, label, value, required, placeholder, name, maximum, minimum, description, errorMessage, valid, visible } = props;
+   const validateState = valid === false ? 'invalid' : 'valid';
+   const error = validateState === 'invalid';
+   const formattedDescription = i18n.formatMessage({ id:`${description}` });
```

Comment:
This will not work. Have to test it?
You have to get the localization id from "props.properties['afs:translationIds']"